docs/react/toggling-devices.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 4
+---
+
+# Toggling devices
+
+Fishjam Cloud provides functionality to control media device streams more dynamically. This includes both soft control, such as muting, and hard control, like physically turning the device on or off.
+
+## Soft Mode - muting and unmuting
```

Comment:
We hid the words 'soft' and 'hard' from our API and replaced them with 'toggleMute' and 'toggleDevice.' Perhaps we should stick to this convention to reduce mental overload.

proposition:
```suggestion
## Muting and unmuting - `toggleMute()`
```

or just:

```suggestion
## Muting and unmuting`
```

Reply:
d